CONTRIBUTING.md

```diff
@@ -0,0 +1,79 @@
+# Contributing to BGPMon
+## Coding standards
+We use the [Code Review Comments](https://github.com/golang/go/wiki/CodeReviewComments) as the basic standard for all of our code.
+In addition to this, we have supplemental standards defined below.
+
+### Gofmt and golint
+All code should be formatted with gofmt and pass golint without any errors. This helps to avoid small formatting errors and lack
+of documentation.
+
+### Import Style
+All imports within the project should be contained within an import() block, even if there's only one. Gofmt will sort imports
+within a group alphabetically. Import groups are separated by a newline. There should only be 3 import groups, appearing in order.
+
+1. Standard library
+2. Local project packages
+3. 3rd party libraries
+
+Some files may not use all of the groups above. Here is an example of the import style:
+
```

Comment:
If you use "\```go" at the start and "\```" at the end of your code snippets then you can get syntax highlighting.

```go
    import (
        "fmt"
        "os"

        "github.com/CSUNetSec/bgpmon/util"

        "github.com/spf13/cobra"
    )
```